### PR TITLE
change: update visual identity

### PR DESCRIPTION
**Pull request by eddieantonio**
- [x] warm logo
 - [x] change colour scheme
 - [ ] ~~add footer graphics~~

### CODE REVIEW

File: src/css/styles.css [hunk 1]
```diff
@@ -413,6 +423,10 @@ html {
   cursor: not-allowed;
 }
 
+/**************************** LANGUAGE SELECTOR *****************************/
+.language-selector {
```
Comment (nienna73) [inline src/css/styles.css:427]: Delete this class?

**Review by nienna73 — COMMENTED** [review]
Just the one comment, it looks good to me after that's addressed.

**Review by nienna73 — APPROVED** [review]
Looks good!

**Comment by eddieantonio** [comment]
Footer graphics will be handled in a separate PR (see #615)

**Pull request merged**